### PR TITLE
feat: add Atmos

### PR DESCRIPTION
**Pull request by mtweeman**
Atmos was already added in #2577, but removed in #2573

### CODE REVIEW

File: docs/registry.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ editLink: false
 | asdf-plugin-manager | [asdf:asdf-community/asdf-plugin-manager](https://github.com/asdf-community/asdf-plugin-manager) |
 | assh | [asdf:zekker6/asdf-assh](https://github.com/zekker6/asdf-assh) |
 | atlas | [asdf:komi1230/asdf-atlas](https://github.com/komi1230/asdf-atlas) |
+| Atmos | [asdf:cloudposse/asdf-atmos](https://github.com/cloudposse/asdf-atmos) |
```
Comment (roele) [inline docs/registry.md]: maybe this should also be lower case

Comment (mtweeman) [reply docs/registry.md]: it seems that all names follow lowercase convention in this file. corrected. thanks for remark!

**Comment by jdx** [comment]
you need to change it here https://github.com/mise-plugins/registry

**Comment by mtweeman** [comment]
> you need to change it here https://github.com/mise-plugins/registry or it will just get overwritten on the next release

I wasn't aware of it. Thanks for letting me know! PR already created (https://github.com/mise-plugins/registry/pull/34)

**Pull request closed without merging**